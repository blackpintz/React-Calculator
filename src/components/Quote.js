import React from 'react';
import styled from 'styled-components';
import Navbar from './Nav';
import '../App.css';

export default function Quote() {
  const PTag = styled.p`
    font-size: 30px;
    width: 90%;
    height: 80vh;
    margin: 0 auto;
    display: flex;
    align-items: center;
  `;
  return (
    <>
      <Navbar />
      <PTag>
        Mathematics is not about numbers, equations, computations or
        algorithms: it is about understanding. - William Paul Thurston
      </PTag>
    </>
  );
}
