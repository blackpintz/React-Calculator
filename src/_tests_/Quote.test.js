import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';
import renderer from 'react-test-renderer';


test('first snapshot test', () => {
  const component = renderer.create(
    <BrowserRouter><Quote /></BrowserRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
