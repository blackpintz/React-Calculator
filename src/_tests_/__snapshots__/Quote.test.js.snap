// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first snapshot test 1`] = `
Array [
  <nav
    className="sc-gsTCUz koSCwl"
  >
    <h2>
      Math Magicians
    </h2>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <span>
      |
    </span>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <span>
      |
    </span>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>,
  <p
    className="sc-bdfBwQ iprfVM"
  >
    Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul Thurston
  </p>,
]
`;
